_data/dtc/random.yml

```diff
@@ -0,0 +1,13 @@
+- Découverte de nos Territoires Communs
```

Comment:
It is not used yet indeed. I just wanted to showcase the words could be defined in YAML and exposed in the frontend cf. #6 

Agreed to move it outside of this PR and make it its own.